samples/snippets/detect_intent_texts_with_location.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python
+
+# Copyright 2017 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""DialogFlow API Detect Intent Python sample to use regional endpoint.
+
+Examples:
+  python detect_intent_texts_with_location.py -h
+  python detect_intent_texts_with_location.py --project-id PROJECT_ID \
+  --location-id LOCATION_ID --session-id SESSION_ID \
+  "hello" "book a meeting room" "Mountain View"
+"""
+
+import argparse
+import uuid
+
+
+# [START dialogflow_detect_intent_text_with_location]
+def detect_intent_texts_with_location(project_id, location_id, session_id, texts, language_code):
+    """Returns the result of detect intent with texts as inputs.
+
+    Using the same `session_id` between requests allows continuation
+    of the conversation."""
+    from google.cloud import dialogflow
+    session_client = dialogflow.SessionsClient(client_options={
+       "api_endpoint": "{location}-dialogflow.googleapis.com".format(location=location_id)})
+
+    session = "projects/{project}/locations/{location}/agent/sessions/{session}".format(
```

Comment:
a bit of a syle choice, but I think you can make all of your format strs into f-strs if you'd like, as our minimum supported version is Python 3.6.

`session = f"projects/{project_id}/locations/{location_id}/agent/sessions/{session_id}"`

Reply:
done